experimental/sql/dialects/ibis_dialect.py

```diff
@@ -119,18 +138,32 @@ class Selection(Operation):
   } {
     // predicates
     ibis.equals() ...
+  } {
+    // projections
+  }
+
+  ibis.selection() {
+    // table
+    ibis.table() ...
+  } {
```

Comment:
nit: can the xdsl parser support more readable custom assembly form ?
e.g.
```
ibis.selection()  
   with table       = { ... }
   with predicates  = { ... } 
   with projections = { ... }
```

Reply:
While that is certainly useful, there is no support for this yet, particularly not autogenerated for each operation. I think it should be possible to define custom printers and parsers, but AFAIK we did not develop very far in that direction yet.

I opened an [issue](https://github.com/xdslproject/xdsl/issues/113) on the min xdsl repo, since i remember us discussing that we want to develop this further. Hopefully, we won't forget with this issue :)